### ISSUE TITLE
Unable to continue loading result for sui-select when using lookup function

### ISSUE THREAD

**Issue by genuinefafa** [edcarroll/ng2-semantic-ui#130]
### ~~Bug~~ Description:

I'm using a `sui-select` taking advantage of the _Search Selection Lookup (Advanced)_ described in [the docs](https://edcarroll.github.io/ng2-semantic-ui/#/components/select).

Since my results can be pretty huge, I'm in need for an event or something like that so it is possible to continue loading after the user reaches close from the end of current results (my API return batches of N options). This is something that select2 does and it is something useful. 

### Link to minimally-working plunker that reproduces the issue:

This is mostly a request/question, so no plunker feels necessary. 

### Versions:

Angular: 4.1.3
ng2-semantic-ui: 0.8.4
Semantic UI: 2.2.10

**Comment by edcarroll** [comment]
To clarify, are you meaning for this to act like a sort of pagination for the results?

**Comment by genuinefafa** [comment]
That's right. As in, my lookup brings: 10 items and total amount of results, it also support request with a starting position. 

Kinda pagination right.

**Comment by rwpino** [comment]
@edcarroll  any update about this new functionality? will be really usefull

**Comment by edcarroll** [comment]
@rwpino nothing yet I'm afraid, I've been very busy and also haven't yet worked out how best to implement this with what is in there currently. Thinking perhaps an observable interface instead. Will update here when things progress 😄

**Comment by genuinefafa** [comment]
someone told me a few days ago _obversables all the way_, so yes, the obervables path seems like the way to go... maybe combine this with #207 ?

**Comment by dev-fullstacktechnology** [comment]
The best option of avoiding your issue is to reduce the number of a search result from the server response (using limit query). Generally searching feature use for quick lookup any detail. And if you have a large number of search result then my suggestion is to go to the separate page.

**Comment by genuinefafa** [comment]
@dev-fullstacktechnology I think it all depends on the use case for the app you are building, mine has many `select` searchable, that can result in many options, going to a different page is the "google way" but not in a big entry form... 

Think of a user registration form with State/City as option, a searchable list will never go to a different page.

Also, select2, the old-mighty searchable input, has pagination and I beleive its API is very well done.

**Comment by dev-fullstacktechnology** [comment]
So you have multiple search options. I know one best example for your case. 
https://www.voucherify.io/

They have multiple search box where you have a drop down on campaign coupon, series, type and other tags.
Among the search, it will display combined result below table in pagination view.

**Comment by genuinefafa** [comment]
I cannot see anything on that site without registering, think I would not like to do. 
Can you share a screenshot?

**Comment by dev-fullstacktechnology** [comment]
Actually, I used that site in my older project of one client, I don't have an account right now. But yeah, you need an account to check what you are looking.

**Comment by genuinefafa** [comment]
I don't really follow you train of thought @dev-fullstacktechnology ... but the thing I'm looking is the same that select2 does, [check this out](https://select2.org/data-sources/ajax), it has a selection input below the title when you can search github repos, also, when you reach the end it continue loading results.

Even better, they claim to have native support for laravel and other frameworks that paginate results. Is that what you were thinking too?
